### PR TITLE
Pull mp22

### PR DESCRIPTION
**Pull request by ajm01**
This PR introduces microprofile 2.2 for boost (boms, poms and new boosters/unittests

### CODE REVIEW

**Comment by scottkurz** [comment]
Note to reviewers: this will be a much smaller diff once #328 is complete.

File: boost-common/src/main/java/boost/common/boosters/AbstractBoosterConfig.java [hunk 1]
```diff
@@ -46,10 +46,14 @@
     public static final String MP_CONFIG_VERSION_13 = "1.3-" + boostVersion + "-" + milestone + "-SNAPSHOT";
```
Comment (scottkurz) [inline boost-common/src/main/java/boost/common/boosters/AbstractBoosterConfig.java]: Can we change "boostVersion to "1.0" (comment left over from PR #328) ?

Comment (ajm01) [reply boost-common/src/main/java/boost/common/boosters/AbstractBoosterConfig.java]: done

File: boost-maven/boost-boms/booster-mp22-bom/pom.xml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		<artifactId>boost-maven-parent</artifactId>
+		<version>0.1.3-SNAPSHOT</version>
+		<relativePath>../..</relativePath>
+	</parent>
+
+	<groupId>boost.boosters</groupId>
+	<artifactId>mp22-bom</artifactId>
+	<packaging>pom</packaging>
+
+	<description>Booster MP21 Feature Bom</description>
```
Comment (scottkurz) [inline boost-maven/boost-boms/booster-mp22-bom/pom.xml]: MP21=>22

File: boost-maven/boost-boms/booster-mp22-bom/pom.xml [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<!-- Copyright (c) 2018 IBM Corporation and others. All rights reserved.
```
Comment (scottkurz) [inline boost-maven/boost-boms/booster-mp22-bom/pom.xml]: 2018->2019

File: boost-maven/boost-boms/boost-mp22-apis-bom/pom.xml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+				<version>${rest-client-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.microprofile.opentracing</groupId>
+				<artifactId>microprofile-opentracing-api</artifactId>
+				<version>${opentracing-version}</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
```
Comment (scottkurz) [inline boost-maven/boost-boms/boost-mp22-apis-bom/pom.xml]: Noting other comments from https://github.com/dev-tools-for-enterprise-java/boost/issues/339  though we can defer until that issue

File: boost-maven/boost-runtimes/runtime-openliberty/src/test/java/io/openliberty/boost/runtimes/boosters/MPFaultToleranceBoosterTest.java [hunk 5]
```diff
@@ -63,4 +63,31 @@ public void testMPFaultToleranceBoosterFeature11() throws Exception {
 
     }
 
+    /**
+     * Test that the mpFaultTolerance-1.2 feature is added to server.xml when the
```
Comment (scottkurz) [inline boost-maven/boost-runtimes/runtime-openliberty/src/test/java/io/openliberty/boost/runtimes/boosters/MPFaultToleranceBoosterTest.java]: Comments are copied without updating versions.

**Pull request merged**